jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/RoadSideUnit.java

```diff
@@ -3,58 +3,57 @@
 import us.dot.its.jpo.ode.model.OdeObject;
 
 public class RoadSideUnit {
-    public static class RSU extends OdeObject {
+   public static class RSU extends OdeObject {
 
-        private static final long serialVersionUID = 3149576493038209597L;
+      private static final long serialVersionUID = 3149576493038209597L;
 
-        private String rsuTarget;
-        private String rsuUsername;
-        private String rsuPassword;
-        private int rsuRetries;
-        private int rsuTimeout;
+      private String rsuTarget;
+      private String rsuUsername;
+      private String rsuPassword;
+      private int rsuRetries;
+      private int rsuTimeout;
 
-        public String getrsuTarget() {
-            return rsuTarget;
-        }
+      public String getrsuTarget() {

```

Comment:
method name should be get**R**suTarget. Please use eclipse to generate getters and setters for you. It is much quicker and will not result in errors like this.